napari/utils/notebook_display.py

```diff
@@ -107,16 +109,27 @@ def _repr_png_(self):
         -------
         In memory binary stream containing PNG screenshot image.
         """
-        from imageio import imsave
+        import PIL
+        from imageio.v3 import imwrite
```

Comment:
This imports fails on minreq. I guess we need to bump the minimal version of `imageio` ?

Reply:
It looks like the minimum version of imageio where you can `import imageio.v3` is version 2.16.0 ([released Feb 14th 2022](https://github.com/imageio/imageio/releases/tag/v2.16.0)).

That does seem recent, but I guess it's better to go with the new API. I've updated the setup.dfg and binder environment.yml files.